keyboards/binepad/bnk9/bnk9.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 binepad (@binepad)
```

Comment:
We have previously rejected a submission implementing the same functionality.

As such, we would require its removal here tool.

Reply:
Understood ... `default` and `via` keymaps are to be kept "pure".  Moving this customisation and supporting files to the vendor `binepad` keymap folder.

Reply:
> We have previously rejected a submission implementing the same functionality.
>
> As such, we would require its removal here too.

The request for for the contents removal, not relocation. Its location is not the issue, so moving it to a vendor keymap wouldn't make a difference.

The issue is about the duplication of functionality that should be generalised and go into core.

Reply:
Fair enough that per-key color should one day be in core.  But this is as the unit is sold - and the intent here is to allow users to compile (or modify) as "per factory" version ... this I've done with other vendors source such as Idobao, Keychron, etc. in the past *(i.e. create a `{vendor_name}` keymap folder)*.  Is there a new posture then to only allow `default` and `via` keymaps on a strict basis (since userspaces is a thing now)?

Reply:
Nevermind ... thank you @keyboard-magpie for pointing out the correct approach to this.  Folder has been renamed.

Reply:
Not nevermind. You still need to remove the custom via content.

Reply:
(lol ...that "nevermind" was directed at my own comment - I had not seen yours when I wrote it.)

Ok .. You have a fair point. I'll remove that keymap entirely and create a vendor userspace.  TY.